client/utils/utils_test.go

```diff
@@ -141,10 +141,10 @@ func TestValidateCmd(t *testing.T) {
 		args    []string
 		wantErr bool
 	}{
-		{"misspelled command", []string{"comission"}, true},
+		{"misspelled command", []string{"commission"}, true},
```

Comment:
this was on purpose according to the above `reason`
```suggestion
		{"misspelled command", []string{"comission"}, true},
```

Reply:
`make format` must have amended it